### ISSUE TITLE
Can't disable Features

### ISSUE THREAD

**Issue by thanhhungtn1** [Schamper/nodebb-plugin-shoutbox#61]
Untick gist and it said saved but not saved, when I comeback gist is still ticked.

**Comment by tedr56** [comment]
Thanks @Schamper for the update.

This occurs in new 0.2.2 as well.

**Comment by belstgut** [comment]
Still happening.
NodeBB v0.7.0-dev.
SB: 0.2.2

**Comment by Ribesg** [comment]
Same for me using NodeBB v0.6.1 and SB 0.2.2.

Here's my ugly workaround while it gets fixed:
````css
#shoutbox-button-gist,
#shoutbox-button-archive,
#shoutbox-button-bug {
  display: none;
}
````

**Comment by Benderwan** [comment]
Also happens with version 0.8

**Comment by Schamper** [comment]
Fixed with v0.3.0.
